### PR TITLE
Request Call placeholder & messages updated

### PR DESCRIPTION
**Pull request by Santhosh-Sellavel**
### Details
Just place holder & phone number error message copy updates for Request a Call Modal.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5945

### Tests & QA Steps
1. Go to Concierge Chat
2. Click on  📞  button in the chat header to open Request a Call Modal
3. New place holder for Phone Number field. `2109400803 `
4. Click `call` without entering any number to check the new error [messages](https://github.com/Expensify/App/issues/5945#issuecomment-957941428).

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
Screenshot 2021-11-03 at 2 43 07 AM

Screenshot 2021-11-03 at 2 31 19 AM

#### Mobile Web
Simulator Screen Shot - iPhone 12 - 2021-11-03 at 02 25 36

Simulator Screen Shot - iPhone 12 - 2021-11-03 at 02 30 20

#### iOS
Simulator Screen Shot - iPhone 12 - 2021-11-03 at 02 24 10

#### Android
Screenshot_1635886131

Screenshot_1635886789

### CODE REVIEW

**Comment by Santhosh-Sellavel** [comment]
@chiragsalian for these messages text alignment looks weird.

**Comment by chiragsalian** [comment]
Good point. I think the error message should maybe be center-aligned. But it's better to rope in design for this so cc-ing @Expensify/design for thoughts/suggestions on how the error message should appear.

**Comment by shawnborton** [comment]
I think the left-alignment is fine as it is.

**Comment by Santhosh-Sellavel** [comment]
@chiragsalian bump!

**Comment by Santhosh-Sellavel** [comment]
Seems the design team is okay with this!

File: src/languages/en.js [hunk 1]
```diff
@@ -476,8 +476,8 @@ export default {
     },
     cameraPermissionsNotGranted: 'Camera permissions not granted',
     messages: {
-        noPhoneNumber: 'Please enter a phone number including the country code e.g +447814266907.',
-        errorMessageInvalidPhone: 'That doesn’t look like a valid phone number. Try again with the country code. e.g. +15005550006',
+        noPhoneNumber: 'Please enter a valid phone number without brackets or dashes. If you\'re outside the US please include your country code, eg. +447782339811',
```
Comment (chiragsalian) [inline src/languages/en.js:479]: Let's not have duplicated text. Let's just keep `errorMessageInvalidPhone` and remove `noPhoneNumber`. Same time let's update the code usages of `noPhoneNumber` to `errorMessageInvalidPhone`.

File: src/languages/es.js [hunk 2]
```diff
@@ -476,8 +476,8 @@ export default {
     },
     cameraPermissionsNotGranted: 'No has habilitado los permisos para acceder a la cámara',
     messages: {
-        noPhoneNumber: 'Por favor escribe un número de teléfono que incluya el código de país e.g +447814266907.',
-        errorMessageInvalidPhone: 'El teléfono no es valido. Inténtalo de nuevo agregando el código de país. P. ej.: +15005550006',
+        noPhoneNumber: 'Por favor, introduce un número de teléfono válido sin paréntesis o guiones. Si reside fuera de Estados Unidos, por favor incluye el prefijo internacional. P. ej. +447782339811',
```
Comment (chiragsalian) [inline src/languages/es.js:479]: Same comment as above applies here.

**Review by chiragsalian — CHANGES_REQUESTED** [review]

**Comment by Santhosh-Sellavel** [comment]
Will update the PR & let you know

File: src/languages/es.js (hunk 2 above)
Comment (Santhosh-Sellavel) [reply src/languages/es.js]: Done

File: src/languages/en.js (hunk 1 above)
Comment (Santhosh-Sellavel) [reply src/languages/en.js]: Done

**Comment by Santhosh-Sellavel** [comment]
Updated PR

**Comment by Santhosh-Sellavel** [comment]
@chiragsalian PR updated

**Review by chiragsalian — APPROVED** [review]
LGTM 👍. We're on a merge freeze at the moment till Monday so I'll merge it then 🙂

**Pull request merged**